requirements-dev.txt

```diff
@@ -1,5 +1,4 @@
 # For `make test`
-ddt
```

Comment:
Incidentally, you can now generate documentation in about 16.5 seconds instead of 17.5 seconds.